lib/api/kuzzle.js

```diff
@@ -194,7 +213,7 @@ function registerErrorHandlers(kuzzle) {
 
   // Remove external listeners (PM2) to avoid other listeners to exit current process
   process.removeAllListeners('unhandledRejection');
-  process.on('unhandledRejection', (err) => {
+  process.on('unhandledRejection', err => {
     console.error(`ERROR: unhandledRejection: ${err.message}`, err.stack); // eslint-disable-line no-console
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 108 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ATrailingComment)